frontend/src/container/NewDashboard/DashboardDescription/index.tsx

```diff
@@ -30,104 +71,430 @@ function DashboardDescription(): JSX.Element {
+										type="text"
+										icon={<ClipboardCopy size={14} />}
+										onClick={(): void => {
+											setCopy(JSON.stringify(selectedData, null, 2));
+											setIsDashbordSettingsOpen(false);
+										}}
+									>
+										Copy as JSON
+									</Button>
+								</section>
+								<section className="delete-dashboard">
+									<DeleteButton
+										createdBy={selectedDashboard?.created_by || ''}
+										name={selectedDashboard?.data.title || ''}
+										id={String(selectedDashboard?.uuid) || ''}
+										isLocked={isDashboardLocked}
+										routeToListPage
+									/>
+								</section>
+							</div>
+						}
+						trigger="click"
+						placement="bottomRight"
+					>
+						<Button icon={<Ellipsis size={14} />} type="text" className="icons" />
+					</Popover>
+					{!isDashboardLocked && editDashboard && (
+						<SettingsDrawer drawerTitle="Dashboard Configuration" />
 					)}
-				</Col>
-				<Col span={14}>
-					<Row justify="end">
-						<DashboardVariableSelection />
-					</Row>
-				</Col>
-				<Col span={1} style={{ textAlign: 'right' }}>
-					{selectedData && (
-						<ShareModal
-							isJSONModalVisible={openDashboardJSON}
-							onToggleHandler={onToggleHandler}
-							selectedData={selectedData}
-						/>
+					{!isDashboardLocked && addPanelPermission && (
+						<Button
+							className="add-panel-btn"
+							onClick={onEmptyWidgetHandler}
+							icon={<PlusOutlined />}
+							type="primary"
+							data-testid="add-panel"
+						>
+							New Panel
+						</Button>
 					)}
+				</div>
+			</section>
+			{(tags?.length || 0) > 0 && (
+				<div className="dashboard-tags">
+					{tags?.map((tag) => (
+						<Tag key={tag} className="tag">
+							{tag}
+						</Tag>
+					))}
+				</div>
+			)}
+			{!isEmpty(description) && (
+				<section className="dashboard-description-section">{description}</section>
+			)}
+			<section className="dashboard-variables">
+				<DashboardVariableSelection />
+			</section>
+			<DashboardGraphSlider />
 
-					<div className="dashboard-actions">
-						{!isDashboardLocked && editDashboard && (
-							<SettingsDrawer drawerTitle={title} />
-						)}
-
-						<Tooltip title="Share" placement="left">
-							<Button
-								className="periscope-btn"
-								style={{ width: '100%' }}
-								onClick={onToggleHandler}
-								icon={<Share2 size={16} />}
-							/>
-						</Tooltip>
-
-						{(isAuthor || role === USER_ROLES.ADMIN) && (
-							<Tooltip
-								placement="left"
-								title={isDashboardLocked ? 'Unlock Dashboard' : 'Lock Dashboard'}
-							>
-								<Button
-									style={{ width: '100%' }}
-									className="periscope-btn"
-									onClick={handleLockDashboardToggle}
-									icon={isDashboardLocked ? <LockFilled /> : <UnlockFilled />}
-								/>
-							</Tooltip>
-						)}
+			<Modal
+				open={isRenameDashboardOpen}
+				title="Rename Dashboard"
+				onOk={(): void => {
+					// handle update dashboard here
```

Comment:
todo? or missed?

Reply:
not needed as we are not using default `ok` and `cancel` buttons , the code is present in the onClickHandler below. will remove this later